tests/helpers/errors.py

```diff
@@ -22,13 +12,6 @@ def get_error_test_cases(errors):
         "title": "could not find job",
         "status": 404
     },
-    'out-of-credit': {
-        "title": "You do not have enough credits",
-        "type": "https://www.rev.ai/api/v1/errors/out-of-credit",
-        "detail": "You have only 60 seconds remaining",
-        "current_balance": 60,
-        "status": 403
-    },
```

Comment:
there isn't actually a problem with the python sdk since it just passes through the api error. I just removed these from our test resources since they are never used in our tests.